### PR TITLE
Added cronjobs for gattaca servers.

### PR DESCRIPTION
**Pull request by pneerincx**

### CODE REVIEW

File: roles/create_group_subfolder_structure/tasks/main.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     state: directory
     group: "{{ item.0.group }}"
     mode: "{{ item.0.mode }}"
-  become: "{% if item.0.owner is defined and item.0.owner | length %}yes{% else %}no{% endif %}"
+  become: "{% if item.0.owner is defined and item.0.owner | length %}true{% else %}fals{% endif %}"
```
Comment (Gerbenvandervries) [inline roles/create_group_subfolder_structure/tasks/main.yml:8]: false?

File: group_vars/gattaca01_convertor/vars.yml [hunk 2]
```diff
@@ -26,14 +26,198 @@ group_subfolder_structure: [
...
+  umcg-labgnkbh:
+    NGS_Automated: 3.4.0
+    NGS_Demultiplexing: 2.4.1
+crontabs:
+  ######################################################################################################################
+  # umcg-atd group
+  ######################################################################################################################
+  - name: benikdown  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-atd-ateambot
+    machines: "groups['user_interface'] | default([]) }}"
```
Comment (scimerman) [inline group_vars/gattaca01_convertor/vars.yml:85]:
```suggestion
    machines: "{{ groups['user_interface'] | default([]) }}"
```

File: group_vars/gattaca01_convertor/vars.yml [hunk 3]
```diff
@@ -26,14 +26,198 @@ group_subfolder_structure: [
...
+  - name: checkSamplesheet  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-gsad-ateambot
+    machines: "groups['user_interface'] | default([]) }}"
+    minute: '*/5'
+    job: /bin/bash -c "{{ configure_env_in_cronjob }};
+         module load GAP/{{ group_module_versions['umcg-gsad']['GAP'] }}-bare;
+         checkSamplesheetWrapper.sh"
+  - name: arrayConversion  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-gsad-ateambot
+    machines: "groups['user_interface'] | default([]) }}"
```
Comment (scimerman) [inline group_vars/gattaca01_convertor/vars.yml:152]:
```suggestion
    machines: "{{ groups['user_interface'] | default([]) }}"
```

File: group_vars/gattaca01_convertor/vars.yml [hunk 4]
```diff
@@ -26,14 +26,198 @@ group_subfolder_structure: [
...
+    minute: '*/5'
+    job: /bin/bash -c "{{ configure_env_in_cronjob }};
+         module load NGS_Automated/{{ group_module_versions['umcg-gsad']['NGS_Automated'] }}-bare;
+         notifications.sh -g umcg-atd -e"
+  ######################################################################################################################
+  # umcg-gap group
+  ######################################################################################################################
+  - name: checkSamplesheet  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-gap-ateambot
+    machines: "groups['user_interface'] | default([]) }}"
```
Comment (scimerman) [inline group_vars/gattaca01_convertor/vars.yml:169]:
```suggestion
    machines: "{{ groups['user_interface'] | default([]) }}"
```

File: group_vars/gattaca01_convertor/vars.yml [hunk 5]
```diff
@@ -26,14 +26,198 @@ group_subfolder_structure: [
...
+    minute: '*/5'
+    job: /bin/bash -c "{{ configure_env_in_cronjob }};
+         module load GAP/{{ group_module_versions['umcg-gap']['GAP'] }}-bare;
+         checkSamplesheetWrapper.sh"
+  - name: arrayConversion  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-gap-ateambot
+    machines: "groups['user_interface'] | default([]) }}"
```
Comment (scimerman) [inline group_vars/gattaca01_convertor/vars.yml:176]:
```suggestion
    machines: "{{ groups['user_interface'] | default([]) }}"
```

File: group_vars/gattaca01_convertor/vars.yml [hunk 6]
```diff
@@ -26,14 +26,198 @@ group_subfolder_structure: [
...
+    minute: '*/5'
+    job: /bin/bash -c "{{ configure_env_in_cronjob }};
+         module load NGS_Automated/{{ group_module_versions['umcg-gap']['NGS_Automated'] }}-bare;
+         notifications.sh -g umcg-atd -e"
+  ######################################################################################################################
+  # umcg-labgnkbh group
+  ######################################################################################################################
+  - name: checkSamplesheet  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-labgnkbh-ateambot
+    machines: "groups['user_interface'] | default([]) }}"
```
Comment (scimerman) [inline group_vars/gattaca01_convertor/vars.yml:193]:
```suggestion
    machines: "{{ groups['user_interface'] | default([]) }}"
```

File: group_vars/gattaca01_convertor/vars.yml [hunk 7]
```diff
@@ -26,14 +26,198 @@ group_subfolder_structure: [
...
+    minute: '*/5'
+    job: /bin/bash -c "{{ configure_env_in_cronjob }};
+         module load ngs-utils/{{ group_module_versions['umcg-labgnkbh']['ngs-utils'] }};
+         checkSamplesheet.sh -s /groups/umcg-labgnkbh/scr01/"
+  - name: demultiplexing  # Unique ID required to update existing cronjob: do not modify.
+    user: umcg-labgnkbh-ateambot
+    machines: "groups['user_interface'] | default([]) }}"
```
Comment (scimerman) [inline group_vars/gattaca01_convertor/vars.yml:200]:
```suggestion
    machines: "{{ groups['user_interface'] | default([]) }}"
```

**Review by Gerbenvandervries — APPROVED** [review]

**Pull request merged**